.github/workflows/byte-web.yml

```
@@ -0,0 +1,52 @@
+  workflow_dispatch:
+
+jobs:
+  byte-web:
+    runs-on: ubuntu-latest
+    env:
+      HOST_PATH: "${{ github.sha }}/"
+      PUBLIC_URL: "https://firmware.onekey.so.com/${{ github.sha }}/"
+      NODE_ENV: "production"
+      CI: false
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)